### PR TITLE
[RW-4285][RISK=NO] Changes to workspace description page -Send data to RDR

### PR DESCRIPTION
**Pull request by NehaBroad**
Send the following data to RDR:
What is the primary purpose of your project?
What are the specific scientific question(s) you intend to study
What are the scientific approaches you plan to use for your study?
What are the anticipated findings from the study? and
Population of interest

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/rdr/RdrExportEnums.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.google.common.collect.BiMap;
 import com.google.common.collect.ImmutableBiMap;
 import org.pmiops.workbench.model.*;
```
Comment (blrubenstein) [inline api/src/main/java/org/pmiops/workbench/rdr/RdrExportEnums.java]: This is existing, but there's a star import here and we're trying to not use those, could you driveby fix?

File: api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java [hunk 2]
```diff
@@ -22,6 +23,7 @@
 import org.pmiops.workbench.db.model.DbWorkspace;
 import org.pmiops.workbench.firecloud.FireCloudService;
 import org.pmiops.workbench.model.RdrEntity;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
 import org.pmiops.workbench.model.UserRole;
 import org.pmiops.workbench.rdr.api.RdrApi;
 import org.pmiops.workbench.rdr.model.*;
```
Comment (blrubenstein) [inline api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java]: Same existing problem but * import.

File: api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java [hunk 3]
```diff
@@ -289,6 +303,72 @@ private RdrWorkspace toRdrWorkspace(DbWorkspace dbWorkspace) {
     return rdrWorkspace;
   }
 
+  WorkspaceDemographic toWorkspaceDemographics(Set<SpecificPopulationEnum> dbPopulationEnumSet) {
+    WorkspaceDemographic demographic = new WorkspaceDemographic();
+
+    demographic.setAccessToCare(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.ACCESS_TO_CARE)
+            ? WorkspaceDemographic.AccessToCareEnum.NOT_EASILY_ACCESS_CARE
+            : WorkspaceDemographic.AccessToCareEnum.UNSET);
```
Comment (blrubenstein) [inline api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java]: What are these setters doing? I', a little confused why it's converting to names that are different?

Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java]: Its because WorkspceDemographic is RDR object so its converting it from what we have in our DB to Workspace Demographic. Our DB store all the population selection as an ArrayList while RDR has different attributes for each cateogory, I can change the name in the yaml to make it easier to understand.
